Xamarin.Forms.Platform.Android/Forms.cs

```diff
@@ -189,8 +225,40 @@ static void SetupInit(Context activity, Assembly resourceAssembly)
 
 			if (!IsInitialized)
 			{
-				// Only need to do this once
-				Registrar.RegisterAll(new[] { typeof(ExportRendererAttribute), typeof(ExportCellAttribute), typeof(ExportImageSourceHandlerAttribute) });
+				if (maybeOptions != null)
```

Comment:
```suggestion
				if (maybeOptions.HasValue)
```
I think it will be more clear